custom/icds/tests/test_custom_rules.py

```diff
@@ -187,35 +188,46 @@ def _set_dob(self, case, dob):
         return CaseAccessors(self.domain).get_case(case.case_id)
 
     def test_person_case_is_under_6_years_old(self):
+        self.assert_person_case_is_under_N_years_old(
+            6, 'ICDS_PERSON_CASE_IS_UNDER_6_YEARS_OLD')
+
+    def test_person_case_is_under_19_years_old(self):
+        self.assert_person_case_is_under_N_years_old(
+            19, 'ICDS_PERSON_CASE_IS_UNDER_19_YEARS_OLD')
+
+    def assert_person_case_is_under_N_years_old(self, n_years, criteria_name):
         rule = _create_empty_rule(self.domain, case_type='person')
-        rule.add_criteria(CustomMatchDefinition, name='ICDS_PERSON_CASE_IS_UNDER_6_YEARS_OLD')
+        rule.add_criteria(CustomMatchDefinition, name=criteria_name)
 
         with _with_case(self.domain, 'person', datetime.utcnow()) as case:
+            dob = datetime(2018, 2, 22, 12, 0)
```

Comment:
this isn't the dob - this is the value for `now` that gets passed when the rule criteria evaluate